enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/components/Summarize/Summarize.tsx

```diff
@@ -0,0 +1,125 @@
+import { useDisclosure } from "@mantine/hooks";
+import { useMemo, useState } from "react";
+import { match } from "ts-pattern";
+
+import { MultiStepPopover } from "embedding-sdk/components/private/util/MultiStepPopover";
+import { AggregationPicker } from "metabase/common/components/AggregationPicker";
+import type { UpdateQueryHookProps } from "metabase/query_builder/hooks/types";
+import {
+  type AggregationItem,
+  getAggregationItems,
+} from "metabase/query_builder/utils/get-aggregation-items";
+import * as Lib from "metabase-lib";
+
+import { useInteractiveQuestionContext } from "../../context";
+import S from "../Picker.module.css";
+import { BadgeList } from "../util/BadgeList";
+import { ToolbarButton } from "../util/ToolbarButton";
+
+export const SummarizeInner = ({
+  query,
+  onQueryChange,
+  stageIndex,
+}: UpdateQueryHookProps) => {
+  const aggregationItems = useMemo(
+    () => getAggregationItems({ query, stageIndex }),
+    [query, stageIndex],
+  );
+
+  const handleRemove = (aggregation: Lib.AggregationClause) => {
+    const nextQuery = Lib.removeClause(query, stageIndex, aggregation);
+    onQueryChange(nextQuery);
+  };
+
+  const label = match(aggregationItems.length)
+    .with(0, () => `Summarize`)
+    .with(1, () => `1 summary`)
+    .otherwise(value => `${value} summaries`);
```

Comment:
Does this need to be localized? e.g.

```suggestion
  const label = match(aggregationItems.length)
    .with(0, () => t`Summarize`)
    .with(1, () => t`1 summary`)
    .otherwise(value => jt`${value} summaries`);
```